### ISSUE TITLE
Some fields are hidden on the has_many view

### ISSUE THREAD

**Issue by adrianthedev** [avo-hq/avo#668]
### Describe the bug
If you go to https://avodemo.herokuapp.com/avo/resources/posts/43, you should see the polymorphic association on the comment's row, but it's hidden.

### System configuration
**Avo version**: `1.21.0`

**Rails version**:

**Ruby version**:

### Screenshots

<img width="1457" alt="image" src="https://user-images.githubusercontent.com/1334409/154353523-a8cc81bf-338b-45b3-b457-5b00bb6c749a.png">
<img width="1452" alt="image" src="https://user-images.githubusercontent.com/1334409/154353544-d4645441-dc1e-4784-aa43-5722c1284d74.png">

**Comment by jony1993** [comment]
Hi @adrianthedev 
As info: For me this is still not working for avo v. 1.22.0

**Comment by adrianthedev** [comment]
Hey, @jony1993 can you share the polymorphic resource configuration. It seems to be working on [avodemo](https://avodemo.herokuapp.com/avo/resources/teams/12). See the reviews at the bottom. They display the user (belongs_to) but hide the team (the polymorphic association). The fact that the team is hidden is a feature, not a bug. All those reviews belong to that team and showing the same team everywhere is redundant.

```ruby
class TeamResource < Avo::BaseResource
  # Other pieces of configuration
  
  field :admin, as: :has_one
  field :members, as: :has_many, through: :memberships
  field :reviews, as: :has_many
end
```

**Comment by jony1993** [comment]
@adrianthedev 
Should we reopen this or is it just related to my setup?

**Comment by adrianthedev** [comment]
Released in `1.22.2`
